test/preview/components/test_whisper_local.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+from unittest.mock import patch, MagicMock
+
+import pytest
+import torch
+
+from haystack.preview.dataclasses import Document
+from haystack.preview.components import LocalWhisperTranscriber
+
+from test.preview.components.base import BaseTestComponent
+
+
+SAMPLES_PATH = Path(__file__).parent.parent / "test_files"
+
+
+class Test_LocalWhisperTranscriber(BaseTestComponent):
+    @pytest.fixture
+    def components(self):
```

Comment:
Where is this used?

Reply:
Right now you can test components initialized in different ways to make sure they're all possible to save to file when added to a pipeline.